code/src/index.css

```diff
@@ -1,13 +1,142 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap");
+
+:root {
+  --black: #000;
+  --white: #fff;
+  --lightgrey: #dcdcdc;
+  --grey: #808080;
+  --pink: #faaab7;
+}
+
+* {
+  font-size: 16px;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto Mono", monospace;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #ffe0e5;
+}
+
+.app-container {
+  margin: 0 auto;
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
+
+.thought-form,
+.thought {
+  border: 1px solid var(--black);
+  box-shadow: 0.4rem 0.4rem var(--black);
+  margin: 1.5rem 1rem;
+}
+
+.thought-form {
+  background-color: var(--lightgrey);
+  padding: 0.5rem 1rem;
+}
+
+.form-input-length + span {
+  font-size: 0.8rem;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.form-text-input {
```

Comment:
Just a tip, but did you know you can use:
resize: none;

so the text-form box will not be able to drag down.